src/common/components/form/fields/input/InputField.tsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
-import { FieldProps } from 'formik';
+import { FieldProps, getIn } from 'formik';
```

Comment:
I see that `getIn` is exported and used in formiks documentation, but it is not documented and I don't think it's a good idea to use undocumented APIs from libraries. Would it be possible to use `lodash/get` or copy this function locally?

Reply:
Its a library thing, `getIn` is some kind of `formik` ulti, errors from field sometime can be array (fieldarray), which kinda complicated to handler with lodash with 1 fallback value only